v3/src/components/codap-component.tsx

```diff
@@ -22,8 +22,9 @@ export interface IProps extends ITileBaseProps {
 }
 
 export const CodapComponent =
-    observer(({ tile, TitleBar, Component, tileEltClass, onCloseTile, onBottomRightPointerDown, onBottomLeftPointerDown,
-      onRightPointerDown, onBottomPointerDown, onLeftPointerDown }: IProps) => {
+    observer(function CodapComponent(
+      { tile, TitleBar, Component, tileEltClass, onCloseTile, onBottomRightPointerDown, onBottomLeftPointerDown,
+      onRightPointerDown, onBottomPointerDown, onLeftPointerDown }: IProps) {
```

Comment:
Evangeline's open PR will conflict with this change, so I think you should back out this change and come back to it later.